content/concept-xrp-ledger-intro.md

```diff
@@ -0,0 +1,88 @@
+
+The supply of XRP available to the general _does_ change due to a few factors:
+
+- Sending transactions in the XRP Ledger destroys a small amount of XRP. Senders choose how much to destroy, with certain minimums based on the expected work of processing the transaction and how busy the network is. If the network is busy, potential transactions that promise to destroy more XRP can cut in front of the transaction queue. This is an anti-spam measure to make it prohibitively expensive to [DDoS](https://en.wikipedia.org/wiki/Denial-of-service_attack) the XRP Ledger network. For more information, see [Transaction Cost](concept-transaction-cost.html).
+- Each account in the XRP Ledger must hold a small amount of XRP in reserve. This is an anti-spam measure to disincentivize making the ledger data occupy too much space. XRP Ledger validators can vote to change the amount of XRP required as a reserve, to compensate for changes in XRP's real-world value. (The last time this happened was in December 2013, when [the reserve requirement decreased from 50 XRP to 20 XRP](https://ripple.com/insights/proposed-change-to-ripple-reserve-requirement-2/).) If the reserve requirement decreases, XRP that was previously locked up by the reserve becomes available again.
+- Ripple (the company) holds a large reserve of XRP in escrow. At the start of each month, 1 billion XRP is released from escrow for Ripple to use. (Ripple uses XRP to incentivize growth in the XRP Ledger ecosystem and sells XRP to institutional investors.) At the end of each month, any remaining XRP the company does not sell or give away is stored into escrow for a 54-month period. For more information on Ripple's escrow policy, see [Ripple Escrows 55 Billion XRP for Supply Predictability](https://ripple.com/insights/ripple-to-place-55-billion-xrp-in-escrow-to-ensure-certainty-into-total-xrp-supply/). For more information on the technical capabilities of the Escrow feature, see [Escrow](concept-escrow.html).
+
+
+## Responsible Software Governance
+[Responsible Software Governance]: #responsible-software-governance
+
+Any piece of software can only be as good as the developers who code and manage it. Ripple employs a team of world-class engineers dedicated full-time to maintaining and improving the XRP Ledger software, especially the core server, `rippled`. The [source code for `rippled`](https://github.com/ripple/rippled/) is available to the public with a permissive open-source license, as are many other parts of the XRP Ledger ecosystem. Ripple engineers follow best practices for software engineering, including:
+
+- A famously strict and thorough code review process
+- Comprehensive code coverage and unit tests
+- Regularly running automated checks for potential vulnerabilities and memory leaks
+- Regularly commissioning external reviews by professional organizations
```

Comment:
Which are published at...?

Reply:
The standard for this type of review is that it's confidential. My (non-lawyer) understanding is that we _legally can't_ release the reports we've commissioned in the past.

Reply:
Hm, so Ripple has to be trusted that these secret reviews by unnamed "professional organizations" are actually taking place. Here's a project that did similar stuff in the open (after all, it is for open source software): https://opencryptoaudit.org

Maybe you could mention the bug bounty program here too?

Another thing that's missing (but out of scope for this description likely) is a public process to get features added, I have yet to see any major contribution to rippled from outside the company. I suspect one of the reasons is that new features are discussed on JIRA and only the code ends up in the open. Something like a CONTRIBUTING file can go a long way.